org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/actions/ToStringTypeProposal.java

```diff
@@ -68,7 +68,7 @@ public void apply(IDocument document) {
 		Display.getDefault().syncExec(() -> {
 			try {
 				IStructuredSelection selection= new StructuredSelection(fType);
-				IWorkbenchSite site= PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage().getActiveEditor().getSite();
+				IWorkbenchSite site= JavaPlugin.getActiveWorkbenchWindow().getActivePage().getActiveEditor().getSite();
```

Comment:
Shouldn't this be JavaPlugin.getActivePage() directly?

Reply:
Done